diff --git a/node_modules/@11ty/eleventy/src/Engines/Liquid.js b/node_modules/@11ty/eleventy/src/Engines/Liquid.js
index 91ac59f..299c5c0 100644
--- a/node_modules/@11ty/eleventy/src/Engines/Liquid.js
+++ b/node_modules/@11ty/eleventy/src/Engines/Liquid.js
@@ -1,5 +1,5 @@
 import moo from "moo";
-import liquidLib, { Tokenizer, evalToken } from "liquidjs";
+import { Liquid as liquidLib, Tokenizer, evalToken } from "liquidjs";
 import { TemplatePath } from "@11ty/eleventy-utils";
 // import debugUtil from "debug";
 
@@ -30,7 +30,7 @@ class Liquid extends TemplateEngine {
 
 	setLibrary(override) {
 		// warning, the include syntax supported here does not exactly match what Jekyll uses.
-		this.liquidLib = override || new liquidLib.Liquid(this.getLiquidOptions());
+		this.liquidLib = override || new liquidLib(this.getLiquidOptions());
 		this.setEngineLib(this.liquidLib);
 
 		this.addFilters(this.config.liquidFilters);
